### PR TITLE
SD_API UI additions for SD workflow  

### PR DESCRIPTION
**Pull request by if-ai**
Updated models, samplers, upscaler hi res fix, denoise strength, minimal changes to the code will add a way to select VAEs and loRas if I find where to get them from the SD_api #batch size is working but I commented out because is duplicating the images, maybe some bug on the seed parameters. https://youtu.be/kAUQ04iNfjI

### CODE REVIEW

**Comment by if-ai** [comment]
I changed the name so is easier to see

File: extensions/sd_api_pictures/script.py [hunk 1]
```diff
@@ -111,31 +129,57 @@ def input_modifier(string):
...
+
+
     payload = {
+
         "prompt": params['prompt_prefix'] + description,
+        "negative_prompt": params['negative_prompt'],
         "seed": params['seed'],
-        "sampler_name": params['sampler_name'],
-        "steps": params['steps'],
+        "sampler_index": params['sampler_index'],
```
Comment (mcmonkey4eva) [inline extensions/sd_api_pictures/script.py:161]: This change is incorrect, `index` is a very old legacy key-name. `sampler_name` was right.

File: extensions/sd_api_pictures/script.py [hunk 2]
```diff
@@ -111,31 +129,57 @@ def input_modifier(string):
...
+        "hr_second_pass_steps": params['hr_second_pass_steps']
+                 
     }
 
     print(f'Prompting the image generator via the API on {params["address"]}...')
     response = requests.post(url=f'{params["address"]}/sdapi/v1/txt2img', json=payload)
     response.raise_for_status()
     r = response.json()
 
+    # This is broken for me please check where I messed up. I am sorry.
```
Comment (mcmonkey4eva) [inline extensions/sd_api_pictures/script.py:182]: wh... what?

File: extensions/sd_api_pictures/script.py [hunk 3]
```diff
@@ -16,23 +17,28 @@
...
-    'prompt_prefix': '(Masterpiece:1.1), detailed, intricate, colorful',
-    'negative_prompt': '(worst quality, low quality:1.3)',
-    'width': 512,
-    'height': 512,
-    'restore_faces': False,
-    'seed': -1,
-    'sampler_name': 'DDIM',
-    'steps': 32,
-    'cfg_scale': 7
+    "address": "http://127.0.0.1:7860",
```
Comment (mcmonkey4eva) [inline extensions/sd_api_pictures/script.py:20]: This set of changes is... odd? Why did you reformat the quotes here?

File: extensions/sd_api_pictures/script.py (hunk 1 above)
Comment (if-ai) [reply extensions/sd_api_pictures/script.py:161]: I was trying to get the samplers to change with name but somehow, they only picked up with index but if name works leave name.

**Pull request closed without merging**